sdk/attestation/azure-security-attestation/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ include(AzureConfigRTTI)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
This is probably not needed

Reply:
@#$@#$ - it snuck in from my cppast prototype.